spot_description/urdf/spot.ros2_control.xacro

```diff
@@ -65,7 +61,7 @@
 
     </xacro:macro>
 
-    <xacro:macro name="spot_ros2_control" params="interface_type has_arm hostname username password tf_prefix">
+    <xacro:macro name="spot_ros2_control" params="interface_type has_arm hostname username password tf_prefix k_q_p k_qd_p">
```

Comment:
nit: Just checking whether the gains should be named to `kp` and `kd` gains.

Reply:
Yep, I renamed them to match spot sdk conventions: https://dev.bostondynamics.com/docs/concepts/joint_control/readme